CHANGES.md

```diff
@@ -0,0 +1,4 @@
+# Changes
+
+## telingo-2.0.0
+  * add extension to handle dynamic formulas 
```

Comment:
This is not true - please revise!

Reply:
I thought this is true? Maybe it is not arbitrary dynamic formulas but such details we can describe in the readme. If it is more than this, please also have a look at the [README.md](https://github.com/potassco/telingo/blob/master/README.md#dynamic-logic).